### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by imavroukakis**

### CODE REVIEW

**Comment by javadev** [comment]
Hi,

Thank you for junit 5 support.

I found some check style errors.

```
/home/runner/work/passninja-java/passninja-java/src/test/java/com/passninja/model/PassTest.java:27:17: error: Name 'pass_is_created_successfully' must match pattern '^[a-z][a-zA-Z0-9]*$'.
/home/runner/work/passninja-java/passninja-java/src/test/java/com/passninja/model/PassTest.java:36:17: error: Name 'pass_can_be_retrieved_by_serial_number' must match pattern '^[a-z][a-zA-Z0-9]*$'.
```

**Comment by imavroukakis** [comment]
@javadev you're welcome, fixed with [`a38ac97` (#2)](https://github.com/flomio/passninja-java/pull/2/commits/a38ac97a25bda927efda5d1ae9ecc06fc4f4b042)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
```
Comment (javadev) [inline .gitignore]: Do we need this long list? It looks like was copied from another project.

File: src/test/java/com/passninja/model/PassTest.java [hunk 2]
```diff
@@ -53,41 +51,35 @@ public void putPass() throws Exception {
...
+    public void should_fail_with_invalid_pass_credentials() {
         Passninja.init("", "");
-        try {
-            Pass.create("Name.a", new HashMap<>());
-            fail("AuthenticationException is expected");
-        } catch (AuthenticationException expected) {
-            expected.getStatusCode();
-            expected.getMessage();
-        }
+        Assertions.assertThrows(AuthenticationException.class, () -> Pass.create("Name.a", new HashMap<>()));
```
Comment (javadev) [inline src/test/java/com/passninja/model/PassTest.java:83]: It looks better!

**Review by javadev — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (imavroukakis) [reply .gitignore]: It's generated via an IntelliJ plugin. It adds ignore rules for non-source artifacts for the following

* Maven
* Eclipse
* Java
* IntelliJ

Comment (javadev) [reply .gitignore]: I think we can exclude `.gitignore` from the pull request.

Or it me be reduced to two strings:
```
target/
.idea/
```

Comment (imavroukakis) [reply .gitignore]: ok! I suggest you keep 
```
.mvn/timing.properties
# https://github.com/takari/maven-wrapper#usage-without-binary-jar
.mvn/wrapper/maven-wrapper.jar
```
as you can add the maven wrapper integration at a later date, allowing local builds without having to download maven first, see https://github.com/takari/maven-wrapper

**Review by javadev — APPROVED** [review]
LGTM

**Pull request merged**